pkg/cluster/internal/providers/docker/network_integration_test.go

```diff
@@ -0,0 +1,81 @@
+// +build !nointegration
```

Comment:
This allows us to run unit tests only without the annoying skip messages. we could drop the skip mechanism entirely, but it's not super onerous for now.